packages/core/upload/admin/src/components/UploadAssetDialog/FromComputerForm.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
Since you are recreating the components why don't you use  the generators to create your components. This way they will all have at least a simple test that you could improve (or not) while developing?
The generators creates component with all the setup to generate an `inlineSnapshot` even with i18n and redux.

Reply:
I forgot about the generators 😓